docs/conf.py

```diff
@@ -55,6 +58,33 @@
 
 html_theme = 'pydata_sphinx_theme'
 html_static_path = ['_static']
+html_css_files = ['css/custom.css']
+html_logo = 'resources/pyfar_logos_fixed_size_pyfar.png'
+html_title = "pyfar"
+html_favicon = '_static/favicon.ico'
+
+
+# -- HTML theme options
+# https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/layout.html
+
+html_theme_options = {
+    "navbar_start": ["navbar-logo"],
+    "navbar_end": ["theme-switcher", "navbar-icon-links"],
```

Comment:
Is it possible to fix the theme to light? Until we don't have extra figures for dark.

Reply:
Yep, it's fixed to light now.